control/xferfcn.py

```diff
@@ -797,7 +797,7 @@ def returnScipySignalLTI(self):
 
         return out
 
-    def _common_den(self, imag_tol=None):
+    def _common_den(self, imag_tol=None, allownonproper=False):
```

Comment:
I think `allow_nonproper` would be more consisten with keyword names used in the toolbox (eg, like imag_tol).